src/peering/setup.py

```diff
@@ -11,7 +11,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.2.1'
+VERSION = '1.0.0'
 
```

Comment:
@Jing-song the initial version for extension should be 1.0.0b1, or 1.0.0 if previous version is stable version

Reply:
please see the doc here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#notes-1

Reply:
Is `0.2.1` a preview version? Do I need to change it to `1.0.0b1`?